### ISSUE TITLE
PCA, TruncatedSVD Transformers

### ISSUE THREAD

**Issue by TomAugspurger** [dask/dask-ml#22]
This would both be useful / not too difficult (hopefully). Can build off the SVD that's in dask.array.

**Comment by TomAugspurger** [comment]
Some accuracy routines in sklearn that'll be useful https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/utils/extmath.py#L211, https://github.com/scikit-learn/scikit-learn/pull/5299
